server/swagger.json

```diff
@@ -2256,9 +2210,652 @@
+          "200": {
+            "description": "An object with links and an array of organizations",
+            "schema": {
+              "$ref": "#/definitions/Organizations"
+            }
+          },
+          "400": {
+            "description": "Failed to load organizations from store",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          },
+          "403": {
+            "description": "Forbidden to access this route",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          },
+          "default": {
+            "description": "Internal server error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": ["organizations", "multiorganization", "multiuser"],
+        "summary": "Create new organization",
+        "parameters": [
+          {
+            "name": "organization",
+            "in": "body",
+            "description": "Organization to create",
+            "schema": {
+              "$ref": "#/definitions/Organization"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Organization successfully created",
+            "headers": {
+              "Location": {
+                "type": "string",
+                "format": "url",
+                "description": "Location of the newly created organization resource"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Organization"
+            }
+          },
+          "400": {
+            "description": "Invalid JSON; or failed to perform operation in data store",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          },
+          "403": {
+            "description": "Forbidden to access this route",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          },
+          "422": {
+            "description": "Invalid data schema for organization",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          },
+          "default": {
+            "description": "Internal server error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/organizations/{id}": {
+      "get": {
+        "tags": ["organizations", "multiorganization", "multiuser"],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "type": "string",
+            "description": "ID of the organization",
+            "required": true
+          }
+        ],
+        "summary": "Retrieve a specific organization",
+        "description": "Organizations are groups of users with various role-based access-control. They are created by SuperAdmins and managed by Admins.",
```

Comment:
This is more the definition of organization; perhaps this should be the definition of the api.